specification/search/resource-manager/Microsoft.Search/preview/2019-10-01-preview/search.json

```diff
@@ -777,6 +828,266 @@
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "The existing shared private link resource definition was successfully updated.",
+            "schema": {
+              "$ref": "#/definitions/SharedPrivateLinkResource"
+            }
+          },
+          "default": {
+            "description": "HTTP 400 (Bad Request): The given shared private link resource name or the shared private link resource definition is invalid; See the error code and message in the response for details. HTTP 404 (Not Found): The subscription, resource group, search service or shared private link resource could not be found. HTTP 409 (Conflict): The specified subscription is disabled.",
+            "schema": {
+              "$ref": "#/definitions/CloudError"
+            }
+          }
+        }
+      },
+      "get": {
+        "tags": [
+          "SharedPrivateLinkResources"
+        ],
+        "operationId": "SharedPrivateLinkResources_Get",
+        "x-ms-examples": {
+          "SharedPrivateLinkResourceGet": {
+            "$ref": "./examples/GetSharedPrivateLinkResource.json"
+          }
+        },
+        "description": "Gets the details of the shared private link resource managed by the Search service in the given resource group.",
+        "externalDocs": {
+          "url": "https://aka.ms/search-manage"
+        },
+        "parameters": [
+          {
+            "$ref": "#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "#/parameters/SearchServiceNameParameter"
+          },
+          {
+            "$ref": "#/parameters/SharedPrivateLinkResourceNameParameter"
+          },
+          {
+            "$ref": "#/parameters/ClientRequestIdParameter"
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "The definition of the shared private link resource was successfully retrieved and is in the response.",
+            "schema": {
+              "$ref": "#/definitions/SharedPrivateLinkResource"
+            }
+          },
+          "default": {
+            "description": "HTTP 404 (Not Found): The subscription, resource group, Search service or shared private link resource could not be found.",
+            "schema": {
+              "$ref": "#/definitions/CloudError"
+            }
+          }
+        }
+      },
+      "delete": {
+        "tags": [
+          "SharedPrivateLinkResources"
+        ],
+        "operationId": "SharedPrivateLinkResources_Delete",
+        "x-ms-examples": {
+          "SharedPrivateLinkResourceDelete": {
+            "$ref": "./examples/DeleteSharedPrivateLinkResource.json"
+          }
+        },
+        "description": "Disconnects the shared private link resource and deletes it from the Search service.",
```

Comment:
This is inaccurate. Simply say: "Deletes the shared private link resource from the Search service."